requirements/base.txt

```diff
@@ -20,3 +20,6 @@ whitenoise==6.1.0
 psycopg2-binary==2.9.3
 djangorestframework==3.12.4
 django-htmx==1.15.0
+gevent==22.10.2  # Updated version of gevent
```

Comment:
If these are in production we dont need these here?

Reply:
i have modified some issues too i thought it may help in this project